src/rtc/h5_prep.py

```diff
@@ -667,39 +701,98 @@ def get_metadata_dict(product_id: str,
         #    ['product_data_access',
         #     'TBD',
         #     'URL to access the product data'],
-        'metadata/processingInformation/parameters/postProcessingFilteringApplied':  # 1.7.4
-            ['post_processing_filtering_applied',
-             STANDARD_RTC_S1_ONLY,
+        'metadata/processingInformation/parameters/' +
```

Comment:
Question: are `+` for readability or to concatenate the strings? If latter, we might not need the `+` sign here.